### PR TITLE
Adding ONNX pipeline and aws utils

### PR DESCRIPTION
**Pull request by ManoharSai2000**

### CODE REVIEW

**Comment by bhavsarpratik** [comment]
@ManoharSai2000 Any ONNX conversion config or code we can add to this?

File: easy_transformers/loader.py [hunk 1]
```diff
@@ -92,6 +95,86 @@ def get_pipeline(
...
+    Args : 
+        model_path : path of the onnx model
+        tokenizer_path : path to the tokenizer (model directory)
+    """
+
+    def __init__(self, model_path, tokenizer_path, label_map):
+        """Load the ONNX model runtime and the tokenizer with label map being the labels to class names mapping"""
+        self.ort_session = ort.InferenceSession(model_path)
+        self.tokenizer = AutoTokenizer.from_pretrained(
+            tokenizer_path, cache_dir="temp", local_files_only=True
```
[the remark this answers is not in the document]
Comment (ManoharSai2000) [reply easy_transformers/loader.py:112]: Done with default

File: easy_transformers/loader.py [hunk 2]
```diff
@@ -92,6 +95,86 @@ def get_pipeline(
...
+        )
+        self.label_map = label_map
+
+    def predict(self, text):
```
[the remark this answers is not in the document]
Comment (ManoharSai2000) [reply easy_transformers/loader.py:116]: Added!

File: easy_transformers/loader.py [hunk 3]
```diff
@@ -92,6 +95,86 @@ def get_pipeline(
...
+
+    """Custom ONNX Runtime Pipeline for Sequence Classification (same as the HuggingFace Pipeline)"""
+
+    """
+    Args : 
+        model_path : path of the onnx model
+        tokenizer_path : path to the tokenizer (model directory)
+    """
+
+    def __init__(self, model_path, tokenizer_path, label_map):
```
[the remark this answers is not in the document]
Comment (ManoharSai2000) [reply easy_transformers/loader.py:108]: Added!

File: easy_transformers/loader.py [hunk 4]
```diff
@@ -92,6 +95,86 @@ def get_pipeline(
...
+    logger.info("Loading Sentiment Model...")
+    model_path = model_name_or_path + "/" + onnx_name
+    tokenizer_path = model_name_or_path
+
+    return ONNXPipelineForSequenceClassification(model_path, tokenizer_path, label_map)
+
+
+class ONNXSequenceClassificationModel:
+    def __init__(
+        self, model_name: str, remote_dir: str, onnx_name: str, label_map: dict
```
[the remark this answers is not in the document]
Comment (ManoharSai2000) [reply easy_transformers/loader.py:158]: Added!

File: easy_transformers/aws_utils.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+    model_path = f"{download_folder}/{model_name}"
+    os.makedirs(model_path, exist_ok=True)
+
+    ## If the folder already contains model files return model_path
+    if os.listdir(model_path):
+        logger.info(f"Model already exists at {model_path} skipping the downloading...")
+        return model_path
+
+    shutil.rmtree(model_path)
```
[the remark this answers is not in the document]
Comment (ManoharSai2000) [reply easy_transformers/aws_utils.py:49]: No It does not delete any root folder!

File: requirements.txt [hunk 6]
```diff
@@ -3,4 +3,7 @@ spacy==2.*
 cachetools==4.*
 python-json-logger==2.0.2
 pytz
-rich
\ No newline at end of file
+rich
+onnxruntime==1.10.0
+cloudpathlib
+cloudpathlib[s3]
```
Comment (bhavsarpratik) [inline requirements.txt:9]: why there are 2 here?

**Review by bhavsarpratik — COMMENTED** [review]
Lets upgrade the version of lib

**Pull request closed without merging**